.github/workflows/integrate-cluster-cmd.yaml

```diff
@@ -64,16 +64,17 @@ jobs:
 
       - name: Collect component log
         working-directory: ${{ env.working-directory }}
-        # if: steps.test.outputs.exit_code != 0
-        if: always()
+        if: steps.test.outputs.exit_code != 0
+        # if: always()
         run: |
           docker exec tiup-cluster-control bash /tiup-cluster/tests/tiup-cluster/script/pull_log.sh /tiup-cluster/logs
           ls ${{ env.working-directory }}
+          cat /tiup-cluster/logs/*.log || true
```

Comment:
Prefer to move this line to the `Output cluster debug log` step because its' name is `Output cluster debug log`...

Reply:
Thanks for your advice